charts/influxdb/templates/cronjob-loadgen.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: influxdb-loadgen
+  namespace: default
+spec:
+  schedule: "*/10 * * * *"
```

Comment:
Should we ever expose this as a field in values? Totally okay if you want to revisit it in the future just seems like it might make sense to have it be parameterized

Reply:
I think we should and I have in the past, forgot to here, thanks for catching that!